### PR TITLE
use total memory for max API requests

### PR DESCRIPTION
**Pull request by harshavardhana**
## Description
use total memory for max API requests

## Motivation and Context
instead of available use max API requests

## How to test this PR?
max_requests would get a bump but  this
is what we need. 

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Optimization (provides speedup with no functional changes)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [ ] Fixes a regression (If yes, please add `commit-id` or `PR #` here)
- [ ] Documentation updated
- [ ] Unit tests added/updated

### CODE REVIEW

**Review by vadmeste — APPROVED** [review]
LGTM

**Comment by minio-trusted** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-large-bucket.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-gateway-s3.sh | :heavy_check_mark: |
| mint-erasure.sh | :heavy_check_mark: |
| mint-dist-erasure.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-compress-encrypt-dist-erasure.sh | :heavy_check_mark: |
| mint-pools.sh | :heavy_check_mark: |
Deleting image on docker hub
Deleting image locally

File: cmd/handler-api.go [hunk 1]
```diff
@@ -86,7 +86,7 @@ func availableMemory() (available uint64) {
 		return
 	}
 
-	available = memStats.Available / 2
+	available = memStats.Total / 2
```
Comment (klauspost) [inline cmd/handler-api.go:89]: Available is not Free:
```
	// RAM available for programs to allocate
	//
	// This value is computed from the kernel specific values.
```

This seems like the correct value.

Comment (harshavardhana) [reply cmd/handler-api.go:89]: Oh I see - then its my mistake @klauspost

Comment (vadmeste) [reply cmd/handler-api.go:89]: Not sure why this is divided by two here. This function is only returning the available memory, the remaining calculation happens outside

**Pull request closed without merging**